src/libs/ReportUtils.ts

```diff
@@ -11,17 +11,20 @@ import {SvgProps} from 'react-native-svg';
 import {ValueOf} from 'type-fest';
 import * as Expensicons from '@components/Icon/Expensicons';
 import * as defaultWorkspaceAvatars from '@components/Icon/WorkspaceDefaultAvatars';
+// eslint-disable-next-line @dword-design/import-alias/prefer-alias
```

Comment:
Do we need this only for the Permissions import? Why for this and not the others?

Reply:
This might not be necessary, I'll double check.

Reply:
Okay, previously importing like `import Permissions from "./Permissions" caused some failing tests but now it doesn't. It might have been a red herring of a different issue I was having. I'll switch this back.